docs_user/modules/proc_migrating-ovn-data.adoc

```diff
@@ -34,9 +30,9 @@ OVSDB_IMAGE=registry.redhat.io/rhosp-dev-preview/openstack-ovn-base-rhel9:18.0
 endif::[]
 SOURCE_OVSDB_IP=172.17.1.49
 ----
-
-The real value of the `SOURCE_OVSDB_IP` can be get from the puppet generated configs:
-
++
+You can get the real value of the `SOURCE_OVSDB_IP` from the puppet generated configurations:
```

Comment:
`real` doesn't seem like the correct term to use unless it has a nuance that I'm unaware of.

How about?

````
You can get the value to set `SOURCE_OVSDB_IP` by querying the puppet-generated configurations:
````